cmake/generic.cmake

```diff
@@ -331,3 +331,52 @@ function(go_test TARGET_NAME)
   add_custom_target(${TARGET_NAME} ALL DEPENDS ${TARGET_NAME}_timestamp ${go_test_DEPS})
   add_test(${TARGET_NAME} ${CMAKE_CURRENT_BINARY_DIR}/${TARGET_NAME})
 endfunction(go_test)
+
+function(pb_cc_library TARGET_NAME)
```

Comment:
we cannot use `protobuf_generate_cpp` directly because we are manually compiling `Protobuf` instead of using `Find(Protobuf)` in our system. If we do not use `Find(Protobuf)`, the `protobuf_generate_cpp` is not defined.